2021q4/avx-bug.adoc

```diff
@@ -0,0 +1,30 @@
+== The AVX bug on amd64
+
+Contact: Konstantin Belousov <kib@FreeBSD.org>
+
+Commit: gitref:73b357be92385cbb70ba19e7023a736af2c6b493[repository=src]
+
+Some CPUs support the so called init optimization for XSAVE, but not all CPUs
+do.  And when they do, 'according to complex internal microarchitectural
+conditions', the optimization might happen or not.  Basically, this
+means that sometimes the CPU does not write all of the state on
+XSAVE and records in xstate_bv that it did not.
+
+On =signal delivery, the OS provides the saved context interrupted by
```

Comment:
Spurious "=" here. (Which may be my fault. I often hit that key instead of backspace and sometimes forget to remove it.)